dissect/target/plugins/os/unix/log/audit.py

```diff
@@ -1,60 +1,82 @@
-import datetime
 import re
+from pathlib import Path
+from typing import Iterator
 
+from dissect.util.ts import from_unix
+
+from dissect.target.helpers.fsutil import open_decompress
 from dissect.target.helpers.record import TargetRecordDescriptor
-from dissect.target.plugin import Plugin, export
+from dissect.target.plugin import Plugin, export, internal
 
 AuditRecord = TargetRecordDescriptor(
     "linux/log/audit",
     [
         ("datetime", "ts"),
-        ("string", "type"),
-        ("string", "id"),
-        ("string", "msg"),
-        ("string", "auditlog"),
+        ("string", "audit_type"),
+        ("varint", "audit_id"),
+        ("string", "message"),
+        ("uri", "source"),
     ],
 )
 
+AUDIT_REGEX = re.compile(r"^type=(?P<audit_type>.*) msg=audit\((?P<ts>.*):(?P<audit_id>.*)\): (?P<message>.*)$")
+
 
 class AuditPlugin(Plugin):
-    def check_compatible(self):
-        path = self.target.fs.path("/var/log/audit/")
+    def __init__(self, target):
+        super().__init__(target)
+        self.log_paths = self.get_log_paths()
+
+    @internal
+    def get_log_paths(self) -> list[Path]:
+        default_config = "/etc/audit/auditd.conf"
+        paths = ["/var/log/audit/audit.log"]
+
+        if (path := self.target.fs.path(default_config)).exists():
+            for line in path.open("rt"):
+                line = line.strip()
+                if not line:
+                    continue
+                if "log_file" in line:
+                    log_file = line.split("=")[-1].strip()
+                    paths.append(log_file)
```

Comment:
```suggestion
                if not line or line.startswith("#") or "log_file" not in line:
                    continue

                log_file = line.split("=")[-1].strip()
                if log_file not in paths:
                    paths.append(log_file)
```
Assuming that this config file uses `#` as comments.

Reply:
Filtering out comments would mean we would miss out on possible previously used log file locations, eg:

```
# log_file = /previously/used/audit.log
log_file = /foo/bar/audit.log
```

Reply:
Fair point, would you suggest the same applies to #127?

Reply:
Good catch, added in #127.